todoman/cli.py

```diff
@@ -113,6 +113,9 @@ def cli(ctx, color, porcelain):
         path for path in glob.iglob(expanduser(config["main"]["path"]))
         if isdir(path)
     ]
+    if len(paths) == 0:
+        print("list is empty\n")
```

Comment:
Please use `click.echo` instead of `print` and also call `ctx.exit(1)` such that todoman doesn't do anything afterwards. Also:

* I prefer something like "No lists found." over "list is empty".
* we should give the user more information: Where to create a new folder (for that we can print out `config["main"]["path"]`.
* `todo --help` doesn't help the user here.